fails to build on master, clientUtils reference undefined

**Describe The Bug**
Can't build at head of master

**Steps To Reproduce**
- clone / checkout master
- run `npm i`

**Expected Behavior**
webpack should not throw an error

**Logs**
```
ERROR in ./scripts/libraryMenu.js 44:0-55
Module not found: Error: Can't resolve './clientUtils' in '/home/jcasper/dev/jellyfin-web/src/scripts'
resolve './clientUtils' in '/home/jcasper/dev/jellyfin-web/src/scripts'
  using description file: /home/jcasper/dev/jellyfin-web/package.json (relative path: ./src/scripts)
    Field 'browser' doesn't contain a valid alias configuration
    using description file: /home/jcasper/dev/jellyfin-web/package.json (relative path: ./src/scripts/clien
tUtils)
```

**System (please complete the following information):**
Arch, npm 8.5.5-1

--- a/src/scripts/libraryMenu.js
+++ b/src/scripts/libraryMenu.js
@@ -14,7 +14,7 @@ import browser from './browser';
 import globalize from './globalize';
 import imageHelper from './imagehelper';
 import { getMenuLinks } from '../scripts/settings/webSettings';
-import Dashboard, { pageClassOn } from './clientUtils';
+import Dashboard, { pageClassOn } from '../utils/dashboard';
 import ServerConnections from '../components/ServerConnections';
 
 import '../elements/emby-button/paper-icon-button-light';

